spring-cloud-dataflow-tasklauncher/spring-cloud-dataflow-tasklauncher-function/src/main/java/org/springframework/cloud/dataflow/tasklauncher/TaskLauncherFunction.java

```diff
@@ -20,11 +20,11 @@
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import java.util.Optional;
+import java.util.function.Consumer;
 import java.util.function.Function;
 
-import org.apache.commons.logging.Log;
-import org.apache.commons.logging.LogFactory;
+import org.slf4j.Logger;
```

Comment:
As we spoke about in the previous PR - we still have not landed on whether  we should be using JCL or SL4J. Lets discuss before starting to switch some pieces to SL4J.

Reply:
Changed back to JCL